components/navbar.vue

```diff
@@ -1,8 +1,40 @@
 <template>
-    <nav>
-        <p>Barra de Navegação</p>
-    </nav>
+    <Toolbar class="!bg-gray-800 !border-gray-800 !rounded-none !py-[2vh] !font-roboto !font-medium !text-xl">
+        <template #start>
+            <div class="flex items-center cursor-pointer" @click="navigateTo('/')">
```

Comment:
You could use a `figure tag` here for better semantics:
https://www.w3schools.com/tags/tag_figure.asp